File: Beginner/C/1009.c
#include <stdio.h>

int main(){

    char nome[10];
    double salario,vendas;

    fgets(nome,sizeof(nome),stdin);
    scanf("%lf", &salario);
    scanf("%lf", &vendas);

    printf("TOTAL = R$ %.2lf\n", salario+(vendas*0.15));
    return 0;
}

File: Beginner/JavaScript/problems/1009.js
export function problem(lines){
    let nome = lines[0];
    let salario = parseFloat(lines[1]);
    let vendas = parseFloat(lines[2]);
    let bonus = 15;

    let salarioTotal = salario + (vendas*(bonus/100));
    console.log(`TOTAL = R$ ${salarioTotal.toFixed(2)}`);
};